sklearn/metrics/classification.py

```diff
@@ -1715,6 +1715,18 @@ def log_loss(y_true, y_pred, eps=1e-15, normalize=True, sample_weight=None,
     y_pred = check_array(y_pred, ensure_2d=False)
     check_consistent_length(y_pred, y_true, sample_weight)
 
+    # GH 10157
```

Comment:
No need to cite the github issue, one will be able to use git blame to find the PR if needed.